config/holes.toml

```diff
@@ -2518,7 +2518,7 @@ preamble = '''
     the end.)
 '''
 
-[Recamán]
+['Recamán']
```

Comment:
Eh I don't think this is necessary. The upcoming TOML v1.1 release supports bare unicode keys and our TOML library supports them so we may as well use them.

Reply:
It looked like that initially. I removed the apostrophes in my initial commit and restored them in my last because the website refused to load. Restoring them seemed to resolve it so I think they are actually necessary, no? Let me do it again to show you what happens in my logs.

```
app-1     | 10:36:18 Listening…					# Removing apostrophes here
app-1     | Killing service
app-1     | ^Csignal: interrupt
app-1     | Starting service
app-1     | panic: toml: expected character ]
app-1     | 
app-1     | goroutine 1 [running]:
app-1     | github.com/code-golf/code-golf/config.unmarshal({0xd664b2?, 0xc0004dfb18?}, {0xc27fa0, 0xc00006ab18})
app-1     |     /go/config/config.go:37 +0x67
app-1     | github.com/code-golf/code-golf/config.init.2()
app-1     |     /go/config/holes.go:63 +0x4c
app-1     | exit status 2					# Restoring apostrophes here
app-1     | Killing service
app-1     | Starting service
app-1     | 10:36:38 Listening…
```
Please have a look at the failed checks from the [first](https://github.com/code-golf/code-golf/actions/runs/9729017101/job/26850400129) and [second](https://github.com/code-golf/code-golf/actions/runs/9729299532/job/26851012047) commit. I can see it there as well.

Reply:
Resolving outdated review conversation. Please review the latest version.